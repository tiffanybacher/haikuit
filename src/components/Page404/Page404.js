import React from 'react';

const Page404 = () => {
  return (
    <section className="Page404">
      <h2>404 Page Not Found</h2>
    </section>
  );
}

export default Page404;