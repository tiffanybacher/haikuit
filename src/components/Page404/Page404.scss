.Page404 {
  background: white;
  padding: 20px;
  text-align: center;
}

@media only screen and (min-width: 768px) {
  .Page404 {
    font-size: 1.1em;
    padding: 30px;
  }
}

@media only screen and (min-width: 1000px) {
  .Page404 {
    font-size: 1.2em;
    padding: 40px;
  }
}

@media only screen and (min-width: 1200px) {
  .Page404 {
    font-size: 1.3em;
    padding: 50px;
  }
}