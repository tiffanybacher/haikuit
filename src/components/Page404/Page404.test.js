import React from 'react';
import { shallow } from 'enzyme';
import Page404 from './Page404';

describe('Page404', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Page404 />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});