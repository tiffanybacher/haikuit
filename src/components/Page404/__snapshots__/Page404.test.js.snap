// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 should match snapshot 1`] = `
<section
  className="Page404"
>
  <h2>
    404 Page Not Found
  </h2>
</section>
`;
